packages/bpk-component-modal/src/bpk-modal-dialog.scss

```diff
@@ -98,9 +98,11 @@
 
   &__dialog-heading {
     margin: 0;
-    font-weight: 700;
 
     @include bpk-text-base;
+
+    // Temporary override as part of BPK-800; to be refactored after BPK-816 is done
+    font-weight: 700; // stylelint-disable-line order/order
```

Comment:
Given that we currently have each text mixin as distinct BEM blocks i.e:
```scss
@include bpk-text-sm;
@include bpk-text-base;
@include bpk-text-lg;
@include bpk-text-xl;
```
as opposed to BEM modifiers i.e:
```scss
@include bpk-text;
@include bpk-text--sm;
@include bpk-text--base;
@include bpk-text--lg;
@include bpk-text--xl;
```
I reckon @tvooo's suggestion of  `@include bpk-text-bold;` makes more sense ... :/